deploy/charts/kaudit/values.yaml

```diff
@@ -70,26 +70,33 @@ resources:
 
 # Azure
 aks:
+  enabled : false
   eventHubName: ""              # Azure EventHubName name (for AKS). If using a non-default ConsumerGroup ($Default), configure it in 'consumer-group-name'.
   eventHubconnectionString: ""  # Azure EventHub connection string (for AKS)
   consumerGroupName: ""         # Azure EventHubName ConsumerGroup name (for AKS), if using a non-default ConsumerGroup (i.e. \$Default).
 
 # GCP
 gke:
+  enabled : false
   projectId: ""                 # GKE-project (for GKE)
   pubsubSubscriptionId: ""      # GKE PubSub subscription ID (only if consuming audit logs via PubSub instead of StackDriver)
   token: ""
   storageType: "pd-ssd"  # Or pd-standard
 
 # EKS or S3
 aws:
+  enabled : true
   kinesisStreamName: ""  # AWS Kinesis stream name (for EKS)
-  region: ""             # AWS Kinesis stream region (for EKS and S3)
+  region:            ""          # AWS Kinesis stream region (for EKS and S3)
+
   accessKeyId: ""        # AWS Kinesis stream credentials (for EKS and S3)
```

Comment:
please add a comment with explanation regarding the use of credentials
key ID and secret OR iam role. but not both
if both are configured, what happens?

Reply:
OK, I haven't tested what will happen if both are included - I can add a test the throws a compeletion error.